bundles/binding/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/GPIOGenericBindingProvider.java

```diff
@@ -176,6 +178,17 @@ public void processBindingConfiguration(String context, Item item, String bindin
                     throw new BindingConfigParseException("Unsupported value for activelow (" + value
                             + ") in configuration string '" + bindingConfig + "'");
                 }
+            } else if (key.compareToIgnoreCase("initialValue") == 0) {
+                if (value.compareToIgnoreCase("high") == 0) {
+                    config.direction = GPIOPin.DIRECTION_OUT_HIGH;
+                } else if (value.compareToIgnoreCase("low") == 0) {
+                    config.direction = GPIOPin.DIRECTION_OUT_LOW;
+                } else {
+                    logger.error("Unsupported value for initialValue (" + value + ") in configuration string '"

```

Comment:
I tried to make changes with minimal modifications. I just copied logging from other parameters and didn't change the way it is used. If it is necessary I'll do this but then only this modification will be with parameterized logging and will differ from the other places in this class.